ui/lib/core/addon/helpers/changelog-url-for.js

```diff
@@ -0,0 +1,33 @@
+import { helper } from '@ember/component/helper';
+
+/*
+This helper returns a url to the changelog for the specified version.
+It assumes that Changelog headers for Vault versions >= 1.4.3 are structured as:
+
+## v1.5.0
+### Month, DD, YYYY
+
+## v1.4.5
+### Month, DD, YYY
+
+etc.
+*/
+
+export function changelogUrlFor([version]) {
```

Comment:
Why do we expect version to be in an array?

Reply:
this is ember weirdness that i don't fully understand, to be honest. ember helpers receive arguments as an array ¯\_(ツ)_/¯

Reply:
That is bizarre. But ok 😂

Reply:
Re: helpers taking arrays: I read a little about this a while ago, and though I can't say I've done this, it's because helpers can take named arguments.  https://guides.emberjs.com/release/components/helper-functions/